packages/dso-toolkit/theme/index.js

```diff
@@ -223,7 +223,7 @@ module.exports = function (options) {
       );
 
       const components = [...entity._app._components.components()._fileTree._items].find(c => c.name === 'componenten');
-      const webComponents = [];
+      const webComponents = ['dso-icon'];
```

Comment:
@tfrijsewijk heeft toegelicht: `dso-icon` moest wel nog als web component geregistreerd worden zodat het niet de markup in andere componenten ging statificeren